cmd/extract.go

```diff
@@ -1,25 +1,63 @@
 package cmd
 
 import (
+	"bufio"
 	"fmt"
 	"os"
+	"path/filepath"
 
 	"github.com/sector-f/jh_extract/nvc"
+	"github.com/spf13/cobra"
 )
 
+func extractCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "extract",
+		Short: "Extract files from a .nvc file",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			arcFilename, _ := cmd.PersistentFlags().GetString("file")
+			pathFilename, _ := cmd.PersistentFlags().GetString("pathlist")
```

Comment:
Yeah, good call

Reply:
Implemented in 7bbca883cd3c5469906b44cd6464ec9083c0c455

It will now just use an empty `[]string` as the pathlist instead of reading from a file if `-p` is missing (or has an empty string as its argument)